include/libopencm3/sam/4l/pm.h

```diff
@@ -26,63 +26,53 @@
 
 /* Main Clock Control MCCTRL Read/Write */
 #define PM_MCCTRL			MMIO32(PM_BASE + 0x000)
+#define PM_MCCTRL_KEY		(PM_UNLOCK_KEY)
 
-/* CPU Clock Select CPUSEL Read/Write */
-#define PM_CPUSEL			MMIO32(PM_BASE + 0x004)
```

Comment:
This is what I'm getting at.  If you're changing this file immediately after commit, just don't commit the first version.  This line above only lives _while_ your branch is being merged.  Just drop it, make the diffs simpler.  Yes. this means you have to rebase and fiddle with your commits, and yes, that can be tedious.